docker-compose.watch.yml

```diff
@@ -6,6 +6,7 @@ services:
             - ./:/srv/pattern-library
             - /srv/pattern-library/node_modules
             - source_assets:/srv/pattern-library/source/assets
+            - ./.container_source_assets:/srv/pattern-library/.container_source_assets
```

Comment:
Changes to  `assets/js/main.js` on the host update within `.container_source_assets` as expected.

CSS does not update as expected. For example when editing `assets/sass/patters/molecules/audio-player.scss` on the host, the change does not propagate to `.container_source_assets/css/audio-player.css` nor to `.container_source_assets/css/all.css`

Reply:
Some sequencing of gulp tasks required. Apart from the wrong folder I'm copying, we need to execute `extract-assets` after any of the other watches has completed their callback.

Reply:
Made a couple of bugfixes, and `extractAssets` should run whenever the interesting tasks are done now.